redirects/k8s.conf

```diff
@@ -0,0 +1,70 @@
+# This file contains all redirects for the Kubernetes Documentation for MinIO
+# This includes redirecting links from legacy docs and the Docs V1 projects
+
+       # Use this as a template
+    #  location <path> {
+    #        return 301 /docs/k8s/<filepath>.html;
+    #  }
+
+
+        location /minio/baremetal/quickstart/k8s.html {
+             return 301 /docs/k8s/operations/index.html;
```

Comment:
Suggest that we go with `/docs/kubernetes/...` for our URL path rather than `/docs/k8s/...`

Reply:
Agreed! Will make that fixup.